### ISSUE TITLE
Sticking vs touching descriptions

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#292]
A11y meeting 9/8/16:

Lets remove "sticking" and "touching" when balloon is on the wall. On the wall will always be classified as "At wall", even when charge is neutral.

Reason is that there are cases when a charged balloon is forced against a wall from the other balloon when it has charge, but there is not enough induced charge in the wall to make it "stick".

**Comment by jessegreenberg** [comment]
It looks like the description for on the sweater can either be "touching" or "sticking" as well. @terracoda mentioned it should only ever be "sticking" or "on" the sweater.

**Comment by jessegreenberg** [comment]
This means that "touching" will never be used in the sim.

**Comment by jessegreenberg** [comment]
Fixed in the above commit, except for the scene summary. The way scene summary descriptions are implemented should be changed so that it pulls directly from BalloonDescriber, WallDescriber, and SweaterDescriber.

**Comment by jessegreenberg** [comment]
Ah, they were different because of capitalization.

**Comment by terracoda** [comment]
@jessegreenberg, that is correct "Touching" will no longer be used in the sim.

And I was just checking the design document, and we already removed "Attractive state" from all the Grab Alerts, so I think this change will only affect scene summary, balloon dynamic descriptions, and balloon releases that end up at the wall.

**Comment by jessegreenberg** [comment]
> this change will only affect scene summary, balloon dynamic descriptions, and balloon releases that end up at the wall.

Sounds good, that is what I am seeing too.  Done in above commits, closing.
